en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,119 @@
+# Construct Statements
+
+The Statement Editor in the Choreo Web Editor is a tool that allows you to construct Ballerina statements in your Choreo Editor even without an in-depth knowledge of the Ballerina Programming Language.
+
+It provides the following:
+
+- Context-based suggestions
+
+- Ballerina libraries (standard libraries and language libraries)
+
+- Pre-defined expression templates
+
+- Documentation for functions and methods with the support to easily configure parameters
```

Comment:
Should we just say:
Documentation for functions and methods to easily configure parameters

Reply:
Changed as suggested.